src/org/ses/android/soap/BaseActivity.java

```diff
@@ -25,17 +30,47 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+    public void logOut()
```

Comment:
Could you add a comment here saying what this does? It's pretty obvious but a quick line or two wouldn't hurt.